dcnm/resource_dcnm_inventory.go

```diff
@@ -789,7 +870,7 @@ func resourceDCNMInventroyDelete(d *schema.ResourceData, m interface{}) error {
 		}
 		d.SetId(strings.Join(leftIps, ","))
 
-		return fmt.Errorf("All switches are not deleted properly")
+		return append(diags, diag.Errorf("All switches are not deleted properly")...)
```

Comment:
probably should be "all" based on other changes.

Reply:
Done